src/transpiler/transpiler.ts

```diff
@@ -51,9 +51,6 @@ export async function transpileFiles(directory: string, outputDir: string, optio
             sourcemap: true,
             dir: outputDir,
             exports: "auto",
-            paths: {
```

Comment:
In `react@18` the production build is conditionally exported based on the value of `NODE_ENV`

Reply:
Also, the `react/cjs/react-jsx-runtime.production.min` file can no longer be resolved because the file is not part of the `react` `package.json#exports`